rails/config/locales/sw/dashboard.sw.yml

```diff
@@ -0,0 +1,53 @@
+en:
```

Comment:
We just have to update this to `sw` in all of the remaining files that are still in English, to be able to merge this PR :)

Reply:
Is there anything needed from my end?

Reply:
Yes, please update `en`  to `sw` at the beginning of each of the remaining files:

* dashboard.sw.yml
* devise.sw.yml
* models.sw.yml

Reply:
I have updated the 3 files.